### PR TITLE
Upgrade to Storybook 5.3.19

### PR DESCRIPTION
**Pull request by Jpsi**
## What does this change?

- Upgrade Storybook from 4.1 to 5.3.19
- Upgrade required dependencies
- Update names of imported modules in stories

## Why are you making this change?

- **Problem**: Storybook did not run
- **Benefit of solution**: Storybook is now running OK
- [**Trello Card**](https://trello.com/c/MsUuFwzC)

## How to make use of this change?

1. Pull the change
1. `[~/code/support-frontend/support-frontend]> rm -rf node_modules` (Remove old packages - first time only)
1. `[~/code/support-frontend/support-frontend]> ./setup.sh` (Reinstall packages - first time only)
1. `[~/code/support-frontend/support-frontend]> yarn storybook` (Run Storybook server)
1. Open Storybook in the browser by going to the localhost URL displayed

### CODE REVIEW

File: support-frontend/Brewfile.lock.json [hunk 1]
```diff
@@ -11,36 +11,36 @@
         }
       },
       "adoptopenjdk8": {
-        "version": "8,252:b09",
+        "version": "8,252:b09.1",
```
Comment (rupertbates) [inline support-frontend/Brewfile.lock.json:14]: Is this required by this PR?

File: support-frontend/Brewfile.lock.json [hunk 2]
```diff
@@ -11,36 +11,36 @@
...
       "guardian/devtools": {
-        "revision": "2d60a611b3029011019b6cb9d3e27d53408dcdfd"
+        "revision": "3b36cf6abae1b19c647beb30fd42ef8b1325da72"
       }
     },
     "brew": {
       "guardian/devtools/dev-nginx": null,
       "awscli": {
-        "version": "2.0.23",
+        "version": "2.0.18",
```
Comment (rupertbates) [inline support-frontend/Brewfile.lock.json:28]: Why are we downgrading this?

File: support-frontend/Brewfile.lock.json [hunk 3]
```diff
@@ -54,11 +54,11 @@
   "system": {
     "macos": {
       "mojave": {
-        "HOMEBREW_VERSION": "2.4.0-67-gb37520c",
+        "HOMEBREW_VERSION": "2.3.0",
```
Comment (rupertbates) [inline support-frontend/Brewfile.lock.json:57]: and this?

**Comment by Jpsi** [comment]
@rupertbates thank you for flagging! Brewfile.lock.json got updated when I ran `[~/code/support-frontend/support-frontend]> ./setup.sh` to install the dependencies in `package.json`.

It seems that Brewfile.lock.json files were introduced in https://github.com/Homebrew/homebrew-bundle/pull/552. Their purpose isn't exactly to "lock" a state and reproduce it (like e.g. yarn.lock), but to add debugging information about the last working setup.

**Pull request merged**